deltachat-ios/AppDelegate.swift

```diff
@@ -598,13 +608,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         UserDefaults.standard.set(slidingTimeframe, forKey: Constants.Keys.notificationTimestamps)
     }
 
-    private func pushToDebugArray(name: String, value: Double) {
+    private func pushToDebugArray(_ value: String) {
+        let name = "notify-fetch-info2"
         let values = UserDefaults.standard.array(forKey: name)
-        var slidingValues = [Double]()
-        if values != nil, let values = values as? [Double] {
-            slidingValues = values.suffix(16)
+        var slidingValues = [String]()
+        if values != nil, let values = values as? [String] {
+            slidingValues = values.suffix(512)
         }
-        slidingValues.append(value)
+        slidingValues.append(value+"@"+DateUtils.getExtendedAbsTimeSpanString(timeStamp: Double(Date().timeIntervalSince1970)))
```

Comment:
minor thing "\(variable)@\(functionCall)" is more swiftish

Reply:
happy to use any other format that makes the time distinguishable when skimming over :) what do you suggest?

Reply:
i switched to `/` which was alrady in use at printing `3`

Reply:
Markdown formatting ate a backslash. What I ment is the formatting above (now correct)